wikimedia/wikipedia-ios
# Image recommendations - Add utility class to style HTML

**Phabricator:**
https://phabricator.wikimedia.org/T356816

### Notes
This PR creates a utility class for converting html strings into NSAttributedString and AttributedString. We need the AttributedString conversion to support attributed SwiftUI text views in the Components framework.

I believe I have covered all of the major needs from our old class in String+Html.swift, though note that I didn't strictly follow it. This utility supports:

- Bold
- Italics
- BoldItalics
- Links
- Lists
- Superscript
- Subscript
- Underline
- Strikethrough

I created some dummy UI to see both UI frameworks in action. Followup PRs for this task will be reusing the article intro view from the `T347121/prototype`, and connecting it to live data.

### Test Steps
1. On Staging or Experimental scheme, launch app.
2. Tap Notifications Center bell icon.
3. The top view is SwiftUI, and the bottom view is UIKit. Confirm styles look correct. Feel free to play with the text in WKImageRecommendationsViewController.swift to test different stylings.

### Screenshots
![screenshot2](https://github.com/wikimedia/wikipedia-ios/assets/3620196/ef558b1a-6d8c-4f44-b3ef-af3843ca859d)
